### ISSUE TITLE
Folder which was full and just renamed is now empty

### ISSUE THREAD

**Issue by pmaniere** [owncloud/core#17358]
1. I renamed a folder. (an important one, of course...) Now, it has no data. 
It seems it deleted the original folder and added a new, empty one. In my opinion, this is a critical bug. 

2. Apart from that, I could not find an easy way to revert to a prior version of the directory in owncloud forums.

Any known info on these topics?

**Comment by LukasReschke** [comment]
Thanks a lot for reporting issues back to us!

We need some more information to properly support you with your issue!

Can I ask you to follow our guidelines for submitting bugs as described here: https://github.com/owncloud/core/blob/master/CONTRIBUTING.md

**Comment by pmaniere** [comment]
Hi Lukas, thx, let me try again :)
But I am missing many items, unfortunately. 

### Steps to reproduce
1. Change name of folder in Finder (Mac Owncloud client v 1.8.1 (build 2335)

### Expected behaviour
Folder name should change. Data contained should not be changed.

### Actual behaviour
Data contained was deleted.

### Server configuration
**Operating system**: 

**Web server:**

**Database:**

**PHP version:**

**ownCloud version:** 

**Updated from an older ownCloud or fresh install:** 

**List of activated apps:** 

**The content of config/config.php:**

```
Insert your config.php content here
(Without the database password, passwordsalt and secret)
```

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** yes/no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

#### LDAP configuration (delete this part if not used)

```
run: sqlite3 data/owncloud.db
then execute:  select * from oc_appconfig where appid='user_ldap';

Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```

### Client configuration
**Browser:** 
- Not relevant for bug.

**Operating system:** 
- Mac OS X 10.10.4

Owncloud client v 1.8.1 (build 2335) 

### Logs
#### Web server error log
```
Insert your webserver log here
```

#### ownCloud log (data/owncloud.log)
```
Insert your ownCloud log here
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log 
c) ...
```

**Comment by LukasReschke** [comment]
Server version?

**Comment by LukasReschke** [comment]
Was it a huge folder? Could be https://github.com/owncloud/core/issues/13391 then which will be fixed in 8.1

**Comment by LukasReschke** [comment]
@PVince81 What would be the best way to restore the data?

**Comment by pmaniere** [comment]
Hi, thanks. I will try to get server version.
Folder was under 20 MB. But critical ones indeed ;)

**Comment by LukasReschke** [comment]
> Hi, thanks. I will try to get server version.

Would be super if you would be able to get the other information as well. Otherwise debugging this will be impossible.

**Comment by PVince81** [comment]
This is very likely to be https://github.com/owncloud/core/issues/13391.
A "./occ files:scan username" should be able to restore the files.

Closing as duplicate.
